assets/js/pages/HostsList/HostsList.jsx

```diff
@@ -129,8 +130,8 @@ function HostsList() {
               </SapSystemLink>,
             ];
           });
-
-          return sidsArray;
+          const extractSid = (item) => item[1]?.props?.children || null;
```

Comment:
Let's use `piping`, right?
in Line 120:  `const sidsArray = uniqBy(sap_systems, getInstanceID).map((instance, index) => {`